spec/spec_helper.rb

```diff
@@ -46,6 +46,7 @@ def clean_database
 end
 
 RSpec.configure do |config|
+  config.expect_with  :minitest, :rspec
```

Comment:
This was a bit tricky - `perform_enqueued_jobs` in the more recent rails calls minitest's `assert` but its not autoloaded in the `mongoid/railtie` path https://github.com/codalio/activity_notification/blob/f0fe033de7382c6808234c65c226db35c9814bc8/spec/rails_app/config/application.rb#L18 so this forces the minitest assertions to load.

I don't know much about mongoid.